src/reference/asciidoc/jdbc.adoc

```diff
@@ -632,6 +632,14 @@ Such connection pools do normally expect that issued connections are closed with
 For this need of an exclusive connection, it is also recommended that a JVM only runs a single `PostgresChannelMessageTableSubscriber` which can be used to register any number of subscriptions.
 ====
 
+==== Transaction support
```

Comment:
I don't think we need a dedicated section id for these paragraphs. It is fully OK to have this info withing that top-level `postgresql-push`.

Reply:
done